csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt

```diff
@@ -14,19 +14,19 @@ id:ID(TagClass)|name:STRING|url:STRING
 sf-003/parquet/31/part.0.parquet
 sf-003/parquet/31/part.1.parquet
 
-creationDate:IGNORE|id:ID(Comment)|locationIP:STRING|browserUsed:STRING|content:STRING|length:STRING
+creationDate:DATETIME|id:ID(Comment)|locationIP:STRING|browserUsed:STRING|content:STRING|length:STRING
 sf-003/parquet/32/part.0.parquet
 sf-003/parquet/32/part.1.parquet
 
-creationDate:IGNORE|id:ID(Forum)|title:STRING
+creationDate:DATETIME|id:ID(Forum)|title:STRING
 sf-003/parquet/33/part.0.parquet
 sf-003/parquet/33/part.1.parquet
 
-creationDate:STRING|id:ID(Person)|firstName:STRING|lastName:STRING|gender:STRING|birthday:STRING|locationIP:STRING|browserUsed:STRING
```

Comment:
We are using this file to test type coercion. Can we leave the types unchanged in this file?

Reply:
as of now, there's no available means to cast a datetime to string (it'll throw an error), so we could either ignore the creationDate (and not follow the original schema) or keep the DATETIME. Thoughts @bhargavivadali ?

Reply:
ok